### ISSUE TITLE
uper  encode and decode error

### ISSUE THREAD

**Issue by CoderHuo** [eerimoq/asn1tools#26]
I found a mistake below.

```
fqdnAsn1 = """
ILP DEFINITIONS AUTOMATIC TAGS ::=
BEGIN
    FQDN ::= VisibleString(FROM ("a".."z" | "A".."Z" | "0".."9" |".-"))(SIZE (1..255))
END
"""

fqdn = asn1tools.compile_string(fqdnAsn1,'uper')
testdata = 'ilp.org'

data_encode = fqdn.encode('FQDN', testdata)
print(binascii.b2a_hex(data_encode))
```

The out print is  b'07d3b382edfcb380'.
That should  be  b'06bb1d41d37b00'
It should be depend on   "X691---- B.2 Extensibility and visibility of constraints in PER".

**Comment by eerimoq** [comment]
Turned out the SIZE and FROM constraints were not implemented for VisibleString in the parser. The UPER codec already had support for them, so the fix was fairly straight forward.

The fix is part of release 0.87.0, available on PyPI.

Hope you find it useful!

**Comment by CoderHuo** [comment]
Thank you very much.
